templates/settings.php

```diff
@@ -41,12 +41,21 @@
 							</p>
 							<p>
 								<label>
-									<input type="checkbox" name="activitypub_enable_blog_user" id="activitypub_enable_blog_user" value="1" <?php echo \checked( '1', \get_option( 'activitypub_enable_blog_user', '0' ) ); ?> />
-									<?php \esc_html_e( 'Enable Blog-Profile', 'activitypub' ); ?>
+									<input type="radio" name="activitypub_actor_mode" id="activitypub_actor_mode" value="<?php echo esc_attr( ACTIVITYPUB_BLOG_MODE ); ?>" <?php echo \checked( ACTIVITYPUB_BLOG_MODE, \get_option( 'activitypub_actor_mode', ACTIVITYPUB_ACTOR_MODE ) ); ?> />
+									<strong><?php \esc_html_e( 'Blog Profile Only', 'activitypub' ); ?></strong>
```

Comment:
English: much less capitalization :)

```suggestion
									<strong><?php \esc_html_e( 'Blog profile only', 'activitypub' ); ?></strong>
```